contracts/OpenOraclePriceData.sol

```diff
@@ -43,9 +41,11 @@ contract OpenOraclePriceData is OpenOracleData {
 
         // Only update if newer than stored, according to source
         Datum storage prior = data[source][key];
-        if (prior.timestamp < timestamp) {
+        if (prior.timestamp < timestamp && timestamp < block.timestamp + 60 minutes) {
```

Comment:
can we change the first clause here to `timestamp > prior.timestamp` that I think reads more obviously

Reply:
yes, done